pom.xml

```diff
@@ -20,6 +20,9 @@
         <sonar.language>java</sonar.language>
         <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
         <checkstyle-maven-plugin.version>3.2.0</checkstyle-maven-plugin.version>
+        <callisto.jpa.rest.version>0.1.11-EAHW-2511-SNAPSHOT</callisto.jpa.rest.version>
+        <micrometer.version>1.10.5</micrometer.version>
```

Comment:
This property can be removed now as micrometer isn't a project dependency anymore

Reply:
This is now removed